.github/workflows/deploy-release.yml

```diff
@@ -27,10 +27,25 @@ on:
         default: true
         required: false
         type: boolean
+  workflow_call:
+    inputs:
+      environment:
+        type: string
+        default: dev
```

Comment:
Sometimes the default has quotes, other times it doesn't. Does it matter if they are there or not?

Reply:
It doesn't matter;
But [the action documentation uses single quotes](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs), so I'll use that here and then consider refactoring other workflows in a future ticket. 
Will notify in Slack thread when this is updated.